packages/components/src/components/ui-shell/_side-nav.scss

```
@@ -157,6 +170,11 @@
     @include expanded($opacity: true, $visibility: true);
   }
 
+  .#{$prefix}--side-nav--ux .#{$prefix}--side-nav__details {
+    opacity: 1;
+    visibility: visible;
```

Due to [the problem with `visibility: visible`](https://jakearchibald.com/2014/visible-undoes-hidden/):

```suggestion
    visibility: inherit;
```